runelite-client/src/main/java/net/runelite/client/plugins/microbot/scurrius/ScurriusConfig.java

```diff
@@ -1,18 +1,161 @@
 package net.runelite.client.plugins.microbot.scurrius;
 
+import lombok.Getter;
+import net.runelite.api.ItemID;
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.Range;
 
 @ConfigGroup("Scurrius")
 public interface ScurriusConfig extends Config {
     @ConfigItem(
-            keyName = "Prioritize",
+            keyName = "foodSelection",
+            name = "Select Food",
+            description = "Select the type of food you want to use",
+            position = 0
+    )
+    default FoodSelection foodSelection() {
+        return FoodSelection.SHARK;
```

Comment:
Why not use Rs2Food here? It will include alot more options to allow the user to pick whatever food they want..

Reply:
Forgot all about it and I changed the food a while ago hah. Updated now to use Rs2Food. There is alot of options now though.